components/dashboard/src/projects/ProjectSettings.tsx

```diff
@@ -80,7 +104,20 @@ export default function ProjectSettingsView() {
 
     return (
         <ProjectSettingsPage project={project}>
-            <Heading2>Prebuilds</Heading2>
+            <Heading2>Project Name</Heading2>
+            <form onSubmit={updateProjectName}>
+                <TextInputField
+                    hint="The name can be up to 32 characters long."
```

Comment:
**nitpick**: Help text look useful, but is needed here all the time? Since we catch this with the validation anyway, thoughts on dropping the help text all together?